src/util.ts

```diff
@@ -282,7 +282,7 @@ export function deleteNestedProperty(obj: any, orderedProps: string[]) {
   if (orderedProps.length === 0) {
     return true;
   }
-  const prop = orderedProps.shift();
+  const prop = orderedProps.shift() as string;
```

Comment:
I think this is cleaner

```suggestion
  const prop = orderedProps.shift()!;
```